lib/testing-helpcenter/data/help_center_main_page_data.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module TestingHelpCentreOnlyoffice
+  # Help Center data for main page links
+  class HelpCenterMainPageData
+    def self.main_page_block_links
+      {
+        'Installation Guides': ['ONLYOFFICE Workspace', 'Workspace', 'Workspace Enterprise Edition', 'Workspace Cloud Service', 'ONLYOFFICE Groups', 'ONLYOFFICE Mail',
```

Comment:
I think this hash is rather complicated and hard to read\format if anything change

Maybe extract it yo `*.yml` file?